config/locales/en.yml

```diff
@@ -2,23 +2,25 @@
 en:
   global_popup_title: "Redmine Advanced Messenger"
   
-  issue_notifications_dialog_go_to_first_one: "Go to first one"
-  issue_notifications_dialog_text_for_current_user: "For %{user}, there are <span id='unread-notifications-count'></span> <span style='color: red'> unread </span> notifications."
-  issue_notifications_dialog_others_tooltip_text: "Unread notifications"
-  issue_notifications_dialog_others_tooltip_text_all_read: "Other users have all messages read."
-  issue_notifications_dialog_status_for_others: "Status for other users"
-  
-  journal_collapse_note_message: "The message is collapsed. Preview: "
-  journal_collapse_note_message_read: "The message is read, but collapsed. Preview: "
-  journal_collapse_note_message_unread: "The message is unread. Please expand it to see its content & mark read."
-  journal_notifications_tooltip_list_title: "%{date} notifications sent (<span style='color:green'>read</span>/<span style='color:red'>unread</span>) to:"
-  journal_notifications_tooltip_list_item_not_viewed: "unread"
-  journal_read: "Read"
-
   label_time_ago: "%{time} ago"
```

Comment:
No, because there are more complex ones:
![image](https://github.com/famiprog/redmine_advanced_messenger/assets/4996054/36d803a7-c857-4404-be4e-ce13b639c3cf)

I needed only: 
![image](https://github.com/famiprog/redmine_advanced_messenger/assets/4996054/f11cbbaa-4a21-407b-8528-16122af3e849)